controllers/ArticlesController.js

```diff
@@ -0,0 +1,26 @@
+const {fetchArticles} = require('../models/ArtcilesModel')
+
+const getArticles = (req, res, next) => {
+    const {article_id} = req.params;
+    if (!article_id || isNaN(article_id)) {
+        return res.status(400).json({ error: 'Invalid or missing article_id' });
+      }
```

Comment:
SQL may handle this for you - saves you writing out the logic. If the query is given an invalid data type it will error